lib/node_modules/@stdlib/repl/lib/resolve_local_scopes.js

```diff
@@ -129,7 +128,7 @@ function resolveScopes( ast ) {
 			break;
 
 		default:
-			throw new Error( format( 'internal error. Unrecognized pattern type: `%s`.', node.type ) );
+			break;
```

Comment:
What was the reason for this change?

Reply:
It was raising an error when the declaration pattern was of type `Literal` (when writing an expression like `(4) =>`. We can just ignore the node if it doesn't match a declaration pattern instead of raising an internal error (which was not even handled in the completer).